app/views/student/bestellingen.php

```diff
@@ -0,0 +1,61 @@
+<?php
+include('./functions/userRoleSystem.php');
+if (! ($userRole == "4") ) {
+    session_unset();
+    header("Refresh: 0; ./index.php");
+}
+// choose between 1: super user,    2: warehouse-admin,     3: financial admin,     4: student
+
+$userId = $_SESSION['id'];
+// Left join for warehouse name or any other needed information
+$sql = "SELECT * FROM `request` LEFT JOIN `warehouse` ON (request.warehouseId = warehouse.id) WHERE $userId = `userId`";
+
+$result = mysqli_query($conn, $sql);
+// record is used for the table top row.
+$record = "";
```

Comment:
this line I would say should not be an echo and it should not be in the view to make it work becease u can make it come out with the rows u can make in the controller of this by using those tags there or if its fixed u can hold it above but using echo's seems kinda useles

Reply:
That is weird this is the old version of the bestellingen page and doesn't have echo's anymore

Reply:
I checked and it's not there anymore in the files